packages/smooth_app/lib/pages/product/product_page.dart

```diff
@@ -470,17 +475,15 @@ class _ProductPageState extends State<ProductPage> {
       // no list to add to
       return;
     }
-    final dynamic target = await showModalBottomSheet<dynamic>(
+    final dynamic target = await showCupertinoModalBottomSheet<dynamic>(
+      expand: false,
       context: context,
-      builder: (final BuildContext context) => Column(
-        children: <Widget>[
-          const Text('Select the destination:'),
-          Wrap(
-            direction: Axis.horizontal,
-            children: children,
-            spacing: 8.0,
-          ),
-        ],
+      backgroundColor: Colors.transparent,
+      bounce: true,
+      barrierColor: Colors.black45,
```

Comment:
I don't like this color: does it work in dark mode?
Unless you really want this color, just stay with the default barrier color, or make sure that it will be ok in both light and dark mode.

Reply:
I have taken the design parameters from the already existing sheets, the barrierColor could not be seen actively anyway, I have now removed this for all sheets where the barrierColor existed.